scripts/india_census/primary_religion_data/test_data/test_Data_Expected.mcf

```diff
@@ -1,810 +1,1304 @@
-Node: dcid:indianCensus/Count_Person_Religion_Total
-description: "Total Population - Total"
-typeOf: dcs:StatisticalVariable
-populationType: schema:Person
-measuredProperty: dcs:indianCensus/Count_Person_Religion_Total
-
-Node: dcid:indianCensus/Count_Person_Religion_Hindu
+Node: dcid:Count_Person_Hindu
+name: "Total Population - Hindu"
```

Comment:
Do we want to curate the name a bit?  (since it will appear on UI surfaces)

1. remove "Total Population" => "Population"
2. "Total Males - Hindu" => "Population - Males - Hindu"
3.  "Male Non-Workers - Religion not stated - Rural" => "Population - Male Non-Workers - Religion ..."
etc.

Reply:
I will remove the name.